CMakeLists.txt

```diff
@@ -419,20 +419,21 @@ if(WITH_OTLP_GRPC
       endif()
     endif()
   endif()
-  # Latest Protobuf imported targets and without legacy module support
-  if(TARGET protobuf::protoc)
-    project_build_tools_get_imported_location(PROTOBUF_PROTOC_EXECUTABLE
-                                              protobuf::protoc)
-    # If protobuf::protoc is not a imported target, then we use the target
-    # directly for fallback
-    if(NOT PROTOBUF_PROTOC_EXECUTABLE)
-      set(PROTOBUF_PROTOC_EXECUTABLE protobuf::protoc)
+  if(NOT PROTOBUF_PROTOC_EXECUTABLE)
+    # Latest Protobuf imported targets and without legacy module support
+    if(TARGET protobuf::protoc)
```

Comment:
Could you please also check and set `PROTOBUF_PROTOC_EXECUTABLE`?
`PROTOBUF_PROTOC_EXECUTABLE` is for the old version of cmake, and now it's `PROTOBUF_PROTOC_EXECUTABLE`.

Reply:
The intent of this change is so that one cat supply `-DPROTOBUF_PROTOC_EXECUTABLE=<path/to/protoc>` when invoking `cmake` on the command line.
I assume you mean I should also make `-DProtobuf_PROTOC_EXECUTABLE` possible, but I don't see a good reason for it. Having to options that do the same thing just creates confusion.

Reply:
For thus project, doesn't this also need precompiled grpc protoc plugin?